CodeEdit/Settings/GeneralSettingsView.swift

```diff
@@ -27,6 +28,13 @@ struct GeneralSettingsView: View {
             .onChange(of: appearance) { tag in
                 tag.applyAppearance()
             }
+
+			Picker("File Icon Style", selection: $fileIconStyle) {
```

Comment:
We should probably start adding these strings to the `Localizable` file so that are ready to be localized!